app/LLM_extractions/extractions.py

```diff
@@ -93,7 +97,26 @@ def extract_information(context: str) -> Optional[Union[dict, str, None]]:
 
         # Filter out keys where the value is None or 'None' (string)
         filtered_ordered_response = {
-            k: v
+            k: (
+                float(v)
```

Comment:
Is there a reason for the explicit type conversions?

Reply:
The LLM model is only able to return the results in string format even when I change the parameter datatypes in the pydantatic model and according to [https://github.com/neurobagel/api/blob/main/app/api/models.py](url) the parameters have these specific datatypes.